email_template_qweb/models/mail_template.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models, tools
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    body_type = fields.Selection(
+        [("jinja2", "Jinja2"), ("qweb", "QWeb")],
```

Comment:
```suggestion
        [("qweb", "QWeb"), ("qweb_view", "QWeb View")],
```

`jinja2` is no longer used in core, it's actually already QWeb..
IMO, if we are to keep this module, the difference between this module and core behaviour is that here we can actually use an `ir.ui.view`, and in core it's raw qweb code

Reply:
Hi, @ivantodorovich I've implemented the changes can you please review them.